src/controllers/pods.ts

```diff
@@ -209,6 +213,7 @@ export default function (server: Hapi.Server, deps: Injector) {
     path: '/pods',
     handler: getPodPrice,
     options: {
+      pre: [{ method: middleware.checkSelfTest, assign: 'selfTest' }],
```

Comment:
It's possible to add the middleware to the server as a whole so that you don't need to add it on every individual endpoint

Reply:
Looks like this is explicitly prohibited: https://github.com/hapijs/hapi/issues/3063#issuecomment-189556345

Reply:
As a pre-handler, maybe, but if you register something as a 'plugin' then it will apply before every route